### PR TITLE
make ChooseMarketActivity

### PR DESCRIPTION
**Pull request by dongcp**

### CODE REVIEW

File: app/src/main/res/layout/nav_header.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:paddingTop="@dimen/nav_header_vertical_spacing"
+        android:src="@android:drawable/sym_def_app_icon"/>
+
+    <TextView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:paddingTop="@dimen/nav_header_vertical_spacing"
+        android:text="Android Studio"
```
Comment (nguyenvantoan20111994) [inline app/src/main/res/layout/nav_header.xml]: text

**Comment by nguyenvantoan20111994** [comment]
ok

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -1,22 +1,29 @@
...
         android:layout_height="wrap_content"
-        android:layout_below="@+id/login_button" />
-</RelativeLayout>
+        android:layout_below="@+id/login_button"/>
+
+    <Button
+        android:id="@+id/button_change_activity"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Change Activity"/>
```
Comment (NguyenXuanTungB) [inline app/src/main/res/layout/activity_main.xml]: hard code

File: app/src/main/res/menu/menu_choose_market.xml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+      xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <item
+        android:id="@+id/action_search"
+        android:icon="@drawable/ic_search"
+        android:title="Search"
```
Comment (NguyenXuanTungB) [inline app/src/main/res/menu/menu_choose_market.xml]: hard code

File: app/src/main/res/menu/menu_navigation_drawer.xml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android">
+    <group android:checkableBehavior="single">
+        <item
+            android:id="@+id/nav_camera"
+            android:icon="@drawable/ic_menu_camera"
+            android:title="Import"/>
```
Comment (NguyenXuanTungB) [inline app/src/main/res/menu/menu_navigation_drawer.xml]: hard code

**Comment by NguyenXuanTungB** [comment]
ok

**Comment by Btnisme123** [comment]
ok

**Pull request merged**